src/components/GameView/CounterDialog.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <choose-whether-to-counter-dialog
+  <ChooseWhetherToCounterDialog
```

Comment:
This is crazy long, not for this PR but maybe we can figure out better naming for some of these.

Reply:
I'm open to suggestions. I tend to favor verbose-and-explicit to concise but any time we can be clear and concise is a win